src/riak_kv_ttaaefs_manager.erl

```diff
@@ -807,41 +1110,57 @@ take_next_workitem([NextAlloc|T], Wants,
 %% configured schedule-needs.
 -spec choose_schedule(schedule_wants()) -> list(allocation()).
 choose_schedule(ScheduleWants) ->
-    [{no_sync, NoSync}, {all_sync, AllSync},
-        {day_sync, DaySync}, {hour_sync, HourSync}] = ScheduleWants,
-    SliceCount = NoSync + AllSync + DaySync + HourSync,
+    [{no_check, NoSync}, {all_check, AllSync},
+        {day_check, DaySync}, {hour_check, HourSync},
+        {range_check, RangeSync}] = ScheduleWants,
+    SliceCount = NoSync + AllSync + DaySync + HourSync + RangeSync,
     Slices = lists:seq(1, SliceCount),
     Allocations = [],
-    lists:sort(choose_schedule(Slices,
-                                Allocations,
-                                {NoSync, AllSync, DaySync, HourSync})).
-
-choose_schedule([], Allocations, {0, 0, 0, 0}) ->
+    lists:sort(
+        choose_schedule(Slices,
+                        Allocations,
+                        {NoSync, AllSync, DaySync, HourSync, RangeSync})).
```

Comment:
naming may be a bit weird here, since you renamed other to NoCheck, AllCheck, etc.

Reply:
Changed to *Check